converter.py

```diff
@@ -103,7 +103,7 @@ def get_dump_name(filename, extension):
             data = processor(data)
         logger.info('Converting {} ({} entries)'.format(
             fname,
-            len(data) if isinstance(data, list) else 1,
+            len(data) if isinstance(data, list) else int(data is not None),
```

Comment:
I guess this should by rights be complemented with something stopping it from writing as well, if `data is None`, otherwise you might have a botched conversion, but the program telling you that no conversion was carried out?

Reply:
you may add that yes.

Reply:
buuut those things that are serialized as arrays of things can be empty lists at least so should not be a truthy check.